src/query/index.ts

```diff
@@ -77,15 +77,13 @@ export const fetchQuery = async <
         ? axios.post(url, data, { headers })
         : axios.get(url, { headers }))
     } catch (error) {
-      // manually capture rate limits
       if (error instanceof AxiosError) {
+        // manually capture rate limits
         if (error.response?.status === 429) {
-          throw new Error('429 too many requests')
-        }
-
-        if (error.response?.data) {
-          throw new Error(`${error.response.data}`)
+          throw new Error('Axios Error: 429 too many requests')
```

Comment:
I think these should be combined but im not sure if there is a dependency on this specific format somewhere